### ISSUE TITLE
Would you give a version number?

### ISSUE THREAD

**Issue by rochefort** [kenjis/ci-phpunit-test#12]
I think that `dev-master` is not good versioning.
I propose to give a version number for each a bunch of commits.

**Comment by kenjis** [comment]
I'm planning to make the first release.

**Comment by RodolfoSilva** [comment]
Create a constant with the version number.

```php
define ('CIPHPUNITTEST_VERSION', '1.0.1');
```
OR

```php
define ('CIPHPUNIT_VERSION', '1.0.1');
```

**Comment by kenjis** [comment]
@RodolfoSilva We don't have to add a constant.
We use Composer and GitHub, so we just tag version numbers.
It will be a release automatically like this: 
* https://github.com/kenjis/codeigniter-composer-installer/releases
* https://packagist.org/packages/kenjis/codeigniter-composer-installer

**Comment by kenjis** [comment]
I've just released v0.1.0: https://github.com/kenjis/ci-phpunit-test/releases

Thanks for feedback you all.

**Comment by rochefort** [comment]
v0.1.0 :congratulations:
